content/insurance/_index.md

```diff
@@ -0,0 +1,31 @@
+# NYIC Tenant Insurance FAQ
+
+So you’ve decided to join NYIC! Great! NYIC requires all tenants and people who will be using tools in the shop to have general liability insurance for their entity listed on the lease. We’ve prepared some general information and FAQ to help guide you through obtaining insurance.
+
+First, a note:
+
+**NYIC can’t tell you what needs to be covered under the policy you’ll need.** You need to insure the activities you’ll be doing and the use of tools you’ll be using in the NYIC space. You don’t need to insure NYIC as a space. NYIC has its own insurance policy should something happen to or in the space. The NYIC policy does not cover you!
+
+### The insurance you will need
+
+A few details about the insurance you’ll need:
+
+- General liability insurance with a $1 million occurrence and $2 million total coverage. This is standard for most general liability insurance.
```

Comment:
i would add "Your policy should cover you for the work you plan on doing in the shop -- be it furniture making, software consulting, etc."